### PR TITLE
Fix home and auth backgrounds

### PR DESCRIPTION
**Pull request by ranquild**
Changes:
- Fixes the background image for auth pages
- Fixes the background color for home and auth pages when whitelabeled

How to test:
- Run Metabase EE 
- Make sure auth and home pages are branded
- Enable whitelabeling
- Make sure auth and home pages are neutral

**Default**
Screenshot 2022-04-22 at 21 25 20
Screenshot 2022-04-22 at 21 25 40

**Whitelabeled**
Screenshot 2022-04-22 at 21 28 21
Screenshot 2022-04-22 at 21 28 33

### CODE REVIEW

**Review by gusaiani — COMMENTED** [review]
@ranquild I believe the decision was to keep the illustration in OS.

**Comment by ranquild** [comment]
@gusaiani It should be also in EE when not whitelabeled, check the message from @kdoh

**Review by gusaiani — APPROVED** [review]
👍👍👍, I'd encourage us to get an ok from @kdoh before merging.

**Pull request merged**
